super_editor/lib/src/infrastructure/super_textfield/_test_tools.dart

```diff
@@ -1,28 +1,27 @@
 import 'package:flutter/services.dart';
 import 'package:flutter/widgets.dart';
 import 'package:flutter_test/flutter_test.dart';
-import 'package:super_editor/src/infrastructure/super_textfield/android/android_textfield.dart';
-import 'package:super_editor/src/infrastructure/super_textfield/ios/ios_textfield.dart';
 
 import 'super_textfield.dart';
 
 extension SuperTextFieldTesting on WidgetTester {
   Future<void> tapAtSuperTextPosition(Finder finder, int offset) async {
-    final match = _findSuperTextField(finder);
+    final fieldFinder = _findInnerPlatformTextField(finder);
+    final match = fieldFinder.evaluate().first.widget;
```

Comment:
Looks like `_findInnerPlatformTextField` throws on multiple matches, so we should be able to do `.single` here.

```suggestion
    final match = fieldFinder.evaluate().single.widget;
```